views/site/partials/index/_featurepoints.php

```diff
@@ -7,7 +7,7 @@
               </div>
               <div class="media-body">
                 <div class="media-heading fast">Fast</div>
-                Yii gives you the maximum functionality by adding the least possible overhead
+                Yii gives you maximum performance by minimizing overhead
```

Comment:
It's adding functionality, not performance.

Reply:
Not that I am biased or anything ( I wrote that sentence :p ) but `Yii gives you maximum performance by minimizing overhead` is nothing like what the original sentence says.
IMO, the original captures exactly what Yii 2 does, and shouldn't be changed.

Reply:
I deliberately wrote `adding the least possible overhead` for emphasis, simply because it is a crucial feature of Yii, not to be overlooked.

Reply:
All right, will remove that. I shortened (and modified) the sentence because I assumed it was about optimal performance, but clearly it's not.